src/App.js

```diff
@@ -1,53 +1,94 @@
 import styled from 'styled-components/macro'
+import {Switch, Route, useHistory} from 'react-router-dom'
+import useGameData from './hooks/useGameData'
+import useForm from './hooks/useForm'
 import Header from './components/layout/Header'
 import Footer from './components/layout/Footer'
-import useGameData from './hooks/useGameData'
-import SaveGameForm from './components/SaveGameForm'
-import GameCardsList from './components/GameCardsList'
-import {Plus, Home} from './components/Icons'
+import Navigation from './components/layout/Navigation'
+import SaveGamePage from './components/SaveGamePage/SaveGamePage'
+import GameCardsPage from './components/GameCardsPage/GameCardsPage'
```

Comment:
Maybe it is better if the pages are not in the components folder, because the pages are actually not components. You could add a pages folder instead.